content/en/docs/components/training/legacy-v1/installation.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Installation"
+description = "How to install the Training Operator"
+weight = 20
++++
+
+This guide describes how to install the Training Operator on your Kubernetes cluster.
+The Training Operator is a lightweight Kubernetes controller that orchestrates the
+appropriate Kubernetes workloads to perform distributed ML training and fine-tuning.
+
+## Prerequisites
+
+These are the minimal requirements to install the Training Operator:
+
+- Kubernetes >= 1.28
+- `kubectl` >= 1.28
+- Python >= 3.7
+
+## Installing the Training Operator
+
+You need to install the Training Operator control plane and Python SDK to create training jobs.
```

Comment:
You call it "control plane" several times, but I think you mean controller? The controller might be installed to the control plane but is it a control plane as we understand them for Kubernetes?

Reply:
The control plane here means similar to Kubernetes control plane: https://kubernetes.io/docs/concepts/overview/components/#control-plane-components
For now, Training Operator control plane contains controller and webhook.